clang/lib/DPCT/APINamesThrust.inc

```diff
@@ -1960,6 +1960,8 @@ CALL_FACTORY_ENTRY("thrust::conj", CALL("std::conj", ARG(0)))
 // thrust::proj
 CALL_FACTORY_ENTRY("thrust::proj", CALL("std::proj", ARG(0)))
 
+// thrust::raw_reference_cast
+CALL_FACTORY_ENTRY("thrust::raw_reference_cast", CALL("dpct::get_raw_reference", ARG(0)))
```

Comment:
should dpct:: namespace be got dynamicly?

Reply:
Currently dpct:: namespace is used default in the migrated code,  while in order to be disabled by option `--use-explicit-namespace=<value>`, I will updated it with `MapNames::getDpctNamespace() + "get_raw_reference"`  instead.